javav2/usecases/creating_sns_sample_app/Readme.md

```diff
@@ -1,4 +1,4 @@
-# Creating a Publish/Subscription Spring Boot Application
+# Creating a Publish/Subscription Web Application that Translates Messages
```

Comment:
Creating a publish/subscription web application that translates messages
* Sentence case

Reply:
Fixed